visual_behavior/data_access/loading.py

```diff
@@ -231,13 +231,18 @@ class BehaviorOphysDataset(BehaviorOphysSession):
                                                 associated with an ophys_experiment_id (single imaging plane)
         """
 
-    def __init__(self, api, include_invalid_rois=False):
+    def __init__(self, api, include_invalid_rois=False, eye_tracking_z_threshold: float = 3.0, eye_tracking_dilation_frames: int = 2):
```

Comment:
Should we consider making the default z_threshold 5 instead of 3? The example you showed me was a pretty compelling case for doing so.

If so, we should also submit an SDK PR to do the same.